opsdroid/connector/github/__init__.py

```diff
@@ -32,16 +32,20 @@ def __init__(self, config, opsdroid=None):
 
     async def connect(self):
         """Connect to GitHub."""
-        url = "{}/user?access_token={}".format(GITHUB_API_URL, self.github_token)
-        async with aiohttp.ClientSession(trust_env=True) as session:
-            response = await session.get(url)
-            if response.status >= 300:
-                _LOGGER.error(_("Error connecting to GitHub: %s."), response.text())
+        url = "{}".format(GITHUB_API_URL)
+        headers = {"Authorization": "token {}".format(self.github_token)}
+        async with aiohttp.ClientSession(trust_env=True, headers=headers) as session:
+            auth_response = await session.get(url)
+            if auth_response.status >= 300:
+                response_text = await auth_response.text()
+                _LOGGER.error(_("Error connecting to GitHub: %s."), response_text)
```

Comment:
What is the purpose of this request? Just to check auth? You could do the same with the `/user` call and reduce the number of requests made when connecting.

Reply:
I thought I initially had issues with doing the single request to `/user`, but I was mistaken. I switched it back to the single request.